vendor/github.com/AudriusButkevicius/pfilter/misc.go

```diff
@@ -0,0 +1,36 @@
+package pfilter
+
+import (
+	"fmt"
+	"net"
+	"sync"
+)
+
+var (
+	maxPacketSize = 2048
```

Comment:
There should be some correlation at least. A random UDP datagram can be up to 65507 bytes in size, which this does not handle. Maybe KCP never creates datagrams that big, but then we should know and have a not about what that maximum size is.